doc/howtos/building-images.rst

```diff
@@ -195,7 +204,7 @@ compressed formats:
 To customize the image format, modify ``local.conf``, adding the variable
 ``OSTRO_VM_IMAGE_TYPES``, set to any combination of the following::
 
-    dsk dsk.xz dsk.vdi dsk.vdi.xz
+    dsk dsk.xz dsk.zip dsk.vdi dsk.vdi.xz dsk.vdi.zip
```

Comment:
On a slightly different note, I noticed earlier it's possible to put in something like "dsk.vdi.bmap" in local.conf and such *.dsk.vdi.bmap file gets created (but no corresponding dsk.vdi)... should this be filed as a bug?